tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -0,0 +1,105 @@
+from datetime import datetime
+
+import os.path
+
+# import pytest
+
+import yaml
+
+
+def read_ert_obs(ert_obs_file_name):
```

Comment:
```suggestion
def read_ert_obs(ert_obs_file_name: str) -> dict:
```

However, maybe even nicer would be to use pathlib for this as well?